Client/Frontend/Browser/Tabs/TabDisplayView.swift

```diff
@@ -5,31 +5,62 @@
 import Common
 import UIKit
 
+struct TabDisplayViewState {
+    var isPrivateTabs: Bool
+    var isInactiveTabEmpty: Bool
+    var isInactiveTabsExpanded = false
+}
+
 class TabDisplayView: UIView,
                       ThemeApplicable,
                       UICollectionViewDataSource,
                       UICollectionViewDelegate,
-                      UICollectionViewDragDelegate,
-                      UICollectionViewDropDelegate {
-    enum UX {}
+                      UICollectionViewDelegateFlowLayout {
+    struct UX {
+        static let cornerRadius: CGFloat = 6.0
+    }
+
+    enum TabDisplaySection: Int, CaseIterable {
+        case inactiveTabs
+    }
+
+    var tabDisplayState: TabDisplayViewState
+    var inactiveTabsSectionManager: InactiveTabsSectionManager
+    var theme: Theme!
```

Comment:
Nit: since it only appears to be used in a few spots it might be nice to avoid this implicitly-unwrapped optional and change to `Theme?`